commitlint.config.ts

```diff
@@ -61,6 +62,53 @@ module.exports = {
 
         {
             rules: {
+                'body-prose': ({body}: {body:any}) => {
+                    let offence = false;
+
+                    // does msg have a body?
+                    if (body !== null) {
+                        let bodyStr = convertAnyToString(body, "body");
+                        
+                        for (let par of bodyStr.trim().split('\n\n')){
+
+                            // It's a paragraph that only consists of a block
+                            if (/^```[^]*```$/.test(par.trim())){
+                                continue;
+                            }
+
+                            par = par.replace(/```[^]*```/g, '').trim();
+
+                            let firstIsUpperCase = par[0].toUpperCase() == par[0];
+                            let firstIsLowerCase = par[0].toLowerCase() == par[0];
+                            let startWithLowerCase = firstIsLowerCase && (!firstIsUpperCase);
```

Comment:
@realmarv how about we create a function called IsUpperCase that calls .toUpperCase and .toLowerCase under the hood? this way this code would be clearer

Reply:
(and other code in other existing rules)

Reply:
Done!